### ISSUE TITLE
[1.8.9] Crash on world load

### ISSUE THREAD

**Issue by aking4125** [Ellpeck/ActuallyAdditions#80]
I would like to report a crash. Not sure which of the two or more is causing it. Removing this mod fixes the problem. Crash occurs when starting a new world.

[crash-2016-03-07_14.33.25-server.txt](https://github.com/Ellpeck/ActuallyAdditions/files/162218/crash-2016-03-07_14.33.25-server.txt)

**Comment by Ellpeck** [comment]
This isn't supposed to be happening.
Are all mods on the newest versions and work with the official RF API?

**Comment by aking4125** [comment]
I created the pack not long before the crash. According to their respective pages. Everything is up to date. As for the Official RF API, not sure what you mean by that one.

**Comment by Ellpeck** [comment]
Does ist crash all the time?

**Comment by aking4125** [comment]
Yeah. For each world I try to create.

**Comment by Ellpeck** [comment]
Can't reproduce.

**Comment by canitzp** [comment]
Can you upload the Modpack as a zip?

**Comment by aking4125** [comment]
Of course you'll close with it without any further investigation. 

@canitzp Owner closed the case.

**Comment by canitzp** [comment]
Yes cause I can't reproduce it, but if your Modpack crashes all the time I'm interessted in it and he isn't

**Comment by Ellpeck** [comment]
I didn't close it "without any further investigation".
We did investigation, we checked every mod, and we tried it out without being able to reproduce.

And stop being rude, thanks.

**Comment by aking4125** [comment]
Nobody is being rude. You just closed it saying you can't reproduce. Which is annoying. You also just stated you "checked every mod" and you didn't get the crash. Perhaps one of mine requires updating. Every thought about giving me some insight?

**Comment by Ellpeck** [comment]
No, because this repository is for Actually Additions.
If you can reproduce the crash with just Actually Additions installed, then repopen the issue.
Otherwise I don't care.

**Comment by canitzp** [comment]
@aking4125 If you give me the Modpack in a zip I look at it and try to find the error, but if you don't give it to me I can't do anythig for your Pack

**Comment by aking4125** [comment]
@Ellpeck So ignoring the obvious fact that your mod will be used in modpacks 90% of the time. Your mod and another (Which I reported to the JEI devs just now to see what is up) is causing the crash however removing only your fixes the problem. I'm not pointing fingers because Modded Minecraft is weird like that. Then you wanna call me rude right after saying "I don't care". What is the point of an issue tracker if you ignore modpacks? 

@canitzp I appreciate the offer. However the official mod author has shown the issue will not be fixed and basically told me to get lost.

**Comment by canitzp** [comment]
@aking4125 I'm the guy that fixes ActAdd and i see that @Ellpeck kciked you from this conversation so if you interested in finding the bug write me on twitter @canitzp

**Comment by Ellpeck** [comment]
Just to be clear:
Just because you have no idea what exactly I tested doesn't mean I didn't test it.

We checked the RF API and the other mods' API Versions and we figured out that, indeed, Actually Additions doesn't appear to be able to cause the crash. That is the reason why we closed the issue.
I just didn't tell you this piece of information because it can easily be shortened down to "can't reproduce".

Still, I don't particularly appreciate you juding the way I handle bugs and issues the (in my opinion, rude) way you did, because I do this for fun. You haven't paid me anything, I haven't paid you anything, we are all doing this for fun and because we enjoy doing it, and if I enjoy doing things the way I do them, you will have to go with that.

Thank you.
